changelogs/changelog.yaml

```diff
@@ -158,7 +158,7 @@ releases:
     release_date: '2024-08-22'
   3.10.1:
     changes:
-      breaking_changes:
+      deprecated_features:
       - mysql_user - the ``user`` alias of the ``name`` argument has been deprecated
```

Comment:
Also 3.10.1 is a bugfix release, it should **not** have deprecations. See the semantic versioning spec: https://semver.org/

Reply:
@felixfontein , I think it was not a deprecation. It's a deprecation warning. The removal will happen in 5.0.
So the version was correct, right?

Reply:
cool, now we know it for future cases, thanks for the info!